docs/mp/aot.adoc

```diff
@@ -26,10 +26,10 @@ include::{rootdir}/includes/mp.adoc[]
 == Contents
 
 - <<Overview, Overview>>
-- <<AOT Supported Modules, AOT Supported Modules>>
+- <<Supported Modules, Supported Modules>>
 
 == Overview
-Helidon applications can be compiled into a native executable using GraalVM
+Helidon applications can be compiled into a native executable using the GraalVM
```

Comment:
this should either be  "GraalVM native image" like before, or it should be "the GraalVM native-image command"